src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -422,6 +422,7 @@ public interface IStartConfiguration
             IAuthProvider AuthProvider { get; set; }
             IEnumerable<IExtension> Extensions { get; set; }
             TaskProcessMode ProcessMode { get; set; }
+            bool HeadlessMode { get; set; }
```

Comment:
Maybe we could do something here, creating objects such that this if() in CheckForProductUpdate will always be true:
```
if(manager.Configuration.DynamoLookUp != null && manager.Configuration.DynamoLookUp.LatestProduct > manager.ProductVersion)
    return;
```